### DISCUSSION TITLE
External keyboard unresponsive when starting i3

### DISCUSSION THREAD

**Discussion by i-pressed-the-button** [i3/i3#6105] [Q&A]
Hi, I'm new to the i3 window manager. I decided to try it out after installing arch (first time with arch as well, might be a bit of a problem with experience? idk).
I have been using an external keyboard all this time as my internal laptop keyboard doesn't work (for some reason, I'm still trying to figure that out). When I was using my command line interface, once I had my keyboard plugged in all was well, it was perfectly usable. But once I start i3 using `startx`, my keyboard stops working. I have not touched the xorg config files other than append `exec i3` to my `~/.xinitrc` file (without that line it would not start), and I have not created any i3 config files either. 
The config wizard window that pops up when I start i3 asks me if I want to have it create a config file, and I hit enter for yes, but there is no response. I have tried other default keys/key combinations as well, but nothing works. The only way to get out of it is to restart my computer by pressing the physical button.
How can this problem be fixed?

**Comment by balpie** [comment]
I an having the same issue, just with my laptop keyboard, so i'd not consider mine external. Didn't manage to fix it yet
